extensions/integration-tests/src/test/index.ts

```diff
@@ -11,7 +11,7 @@ const suite = 'Extension Integration Tests';
 const mochaOptions: any = {
 	ui: 'tdd',
 	useColors: true,
-	timeout: 60000
+	timeout: 300000
```

Comment:
@Charles-Gagnon @alanrenmsft let me know if you have any concern here. The timeout for integration tests is only 1 min and causing some to timeout. I increased it to 5 min here.

Reply:
Why not just apply the longer timeout to the tests/suites that need it instead of all of them? That'd be preferable

Reply:
how can I set that for just one integration test? 

---
In reply to: [537873156](https://github.com/microsoft/azuredatastudio/pull/13649#discussion_r537873156) [](ancestors = 537873156)

Reply:
https://mochajs.org/#timeouts

Reply:
thanks

---
In reply to: [537894428](https://github.com/microsoft/azuredatastudio/pull/13649#discussion_r537894428) [](ancestors = 537894428)